1calculator.c

```diff
@@ -0,0 +1,215 @@
+#include <stdio.h>
+#include <stdlib.h>
+
+// Max size for expression
+#define MAX_SIZE 1000
+
+int is_space(char ch) {
+    if(ch == ' ') return 1;
+    return 0;
+}
+
+int is_digit(char ch) {
+    if(ch >= '0' && ch <= '9')return 1;
+    return 0;
+}
+
+// Definition for operator stack
+typedef struct
+{
+    int top;
+    char data[MAX_SIZE];
+} operator;
+
+// Definition for operand stack
+typedef struct
+{
+    int top;
+    int data[MAX_SIZE];
+} operand;
+
+// Funtion associated with operator stack
+void push_operator(operator* stk, char val)
```

Comment:
you should not use short forms while naming. 
Do specify the meaningful name. 
Its unclear what "stk" and "val" means ?